osmomath/go.mod

```diff
@@ -0,0 +1,89 @@
+module github.com/osmosis-labs/osmosis/osmomath
```

Comment:
Can we suffix this with a version?

I think not having a version might contribute to import cycles and other import problems in the future

Reply:
This is not 1.0 yet. We only need version suffixes once theres a 2.0

Reply:
This opens up a discussion of the semantics of versioning for these submodules. What sort of changes are needed for a v2? is it any api breaking changes?

I think this depends on wether we have external users or not, but I'm almost leaning towards that projects importing this should target a specific commit in their go.mod. That way we don't have to worry about breaking changes for external users.